spec/nagareboshi/configuration_spec.rb

```diff
@@ -7,6 +7,7 @@
     let(:configuration) { Nagareboshi::Configuration.new }
     it { expect(configuration.host).to eq "pubsubhubbub.appspot.com" }
     it { expect(configuration.send).to be_falsy }
-    it { expect(configuration.use_ssl).to be_truthy }
+    it { expect(configuration.use_ssl).to be_falsy }
```

Comment:
Fix expected value because default `use_ssl` on `Nagareboshi::Configuration` is `false`.